src/corehost/cli/hostpolicy/hostpolicy.cpp

```diff
@@ -234,40 +74,22 @@ int run_as_lib(
             return StatusCode::Success;
         }
 
-        prepare_to_run_t prep{ hostpolicy_init, args, false /* breadcrumbs_enabled */ };
-
         // Build variables for CoreCLR instantiation
```

Comment:
Update the comment - while context init does more or less what it says, I don't think it should be mentioned here.